config/routes.rb

```diff
@@ -1,6 +1,7 @@
 OneBody::Application.routes.draw do
 
-  root to: redirect('/stream')
+  if Setting.find_by_name('Stream').value then root to: redirect('/stream')
+  else root to: redirect('/search') end
```

Comment:
@AndyGauge does this work with multisite? I'd be a little surprised if it does. I would expect the routes file to be read once at start up and not refreshed on each request.

If you're not sure, I can test it myself.